python/cdp/evm_client.py

```diff
@@ -302,9 +307,52 @@ async def sign_transaction(
             x_idempotency_key=idempotency_key,
         )
 
-    async def send_transaction(self, transaction: str, network: str):
-        """Send an EVM transaction. Currently unimplemented."""
-        raise NotImplementedError("Sending transactions is not yet implemented")
+    async def send_transaction(
+        self,
+        address: str,
+        transaction: str | TransactionDictType,
+        network: str,
+        idempotency_key: str | None = None,
+    ):
+        """Send an EVM transaction.
+
+        Args:
+            address (str): The address of the account.
+            transaction (str | TransactionDictType): The transaction to send.
+            network (str): The network.
+            idempotency_key (str, optional): The idempotency key. Defaults to None.
+
+        Returns:
+            str: The transaction hash.
+
+        """
+        if isinstance(transaction, str):
+            return (
+                await self.api_clients.evm_accounts.send_evm_transaction(
+                    address=address,
+                    send_evm_transaction_request=SendEvmTransactionRequest(
+                        transaction=transaction, network=network
+                    ),
+                    x_idempotency_key=idempotency_key,
+                )
+            ).transaction_hash
+        else:
+            typed_tx = TypedTransaction.from_dict(transaction)
+            typed_tx.transaction.dictionary["v"] = 0
+            typed_tx.transaction.dictionary["r"] = 0
+            typed_tx.transaction.dictionary["s"] = 0
+            payload = typed_tx.transaction.payload()
+            serialized_tx = bytes([typed_tx.transaction_type]) + payload
```

Comment:
Shouldn't we enforce that the provided transaction is eip1559 (rather than trusting that the client passes one)?